cap/modules/access/views.py

```diff
@@ -38,12 +38,22 @@ def identity_loader_session():
     try:
         identity = Identity(
             session['identity.id'], session['identity.auth_type'])
-        identity.provides = session['identity.provides']
+        # identity.provides = session['identity.provides']
+        identity.provides = get_provides_god()
```

Comment:
This trick is temporal. It allows to use all the roles possible regardless the user that you are using, which it is pretty useful to avoid the OAuth identification.